### PR TITLE
feat: Update protoc and regenerate with IoT Core events

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

File: generate-code.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@
 set -e
 
 echo "~ START"
-PROTOBUF_VERSION=3.19.0
+PROTOBUF_VERSION=21.6
```
Comment (amanda-tarafa) [inline generate-code.sh:14]: Should this be
```suggestion
PROTOBUF_VERSION=3.21.6
```

File: validate-schema.sh [hunk 2]
```diff
@@ -35,7 +35,7 @@ cd $(dirname $(readlink -f "$0"))
 rm -rf tmp
 mkdir tmp
 
-PROTOBUF_VERSION=3.19.0
+PROTOBUF_VERSION=21.6
```
Comment (amanda-tarafa) [inline validate-schema.sh:38]: And same here?
```suggestion
PROTOBUF_VERSION=3.21.6
```

File: generate-code.sh (hunk 1 above)
Comment (jskeet) [reply generate-code.sh:14]: Nope. See the commit comment - protobuf has changed its versioning. (This is protobuf project versioning, not NuGet package versioning which we use elsewhere.)

**Review by amanda-tarafa — APPROVED** [review]
LGTM, I looked at all commits together so missed the comment. Sorry about that.

**Comment by jskeet** [comment]
No problem :)

**Pull request merged**